jupyter_book/config.py

```diff
@@ -159,6 +159,12 @@ def get_final_config(
         "latex_individualpages": cli_config.pop("latex_individualpages"),
     }
 
+    if (
+        "use_jupyterbook_latex" in sphinx_config
```

Comment:
won't `use_jupyterbook_latex` always be in `sphinx_config` since it's a default? And in that case we could probably just do `sphinx_config.get("use_jupyterbook_latex")` no?

Reply:
yup. you are right. have made the changes.